Responsive Styles

chriswynn opened the pull request:
http://56e2d808d6865d2947000035.rrrhody.netlify.com/

sethkrasnianski commented:
The spaces aren't imperative actually. But once the logos are broken into a separate commit :+1: from me. @lupinskij?

Commit:
Add new contributor logos

Commit:
Update lifecycle logo

Commit:
Update mojotech logo

Commit:
Update rrrhody logo

Commit:
Rename lint task appropriately

Commit:
Correctly name js path variable

Commit:
Added stylint task

Commit:
Updated responsive styles

style update

Testimonials style update

Update styles

Add link to the footer

Added spaces to responsive style

Commit:
Rename style task to jscs

Commit:
Add stylint config file

sethkrasnianski commented:
@mordecai174 @lupinskij :eyes:
